src/main/java/duke/Main.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.data.MainWindow;
+import javafx.application.Application;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+
+import javafx.scene.layout.AnchorPane;
+
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+
+import duke.data.Duke;
+
+/**
+ * Main file to run the program.
+ */
+public class Main extends Application {
+
+    /**
+     * Start the GUI.
+     * @param stage
```

Comment:
According to the coding standards there should be an empty line between the description and parameters section!

Reply:
Perhaps a description could have been given for the parameter 'stage' for others to understand this method more easily!